nextflow.config

```diff
@@ -292,7 +292,7 @@ manifest {
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
     version         = '2.1.0'
-    doi             = ''
+    doi             = 'https://doi.org/10.5281/zenodo.3240506'
```

Comment:
I don't think this DOI makes sense. Shouldn't it be a pipeline specific one, like the one for RNAseq ([https://zenodo.org/records/13769441](https://zenodo.org/records/13769441))?

Reply:
Is the pipeline specific one, but in the last release for some reason everything is wrong (authors, version number,...) I was going to try to fix it in the zenodo site, if possible.

Reply:
Ah I see it now, thanks